### PR TITLE
Feat/5 refactor loggly code

### PR DESCRIPTION
**Pull request by TianyiLi**
use typescript private method create class private attribute
fix type declaration incorrect
use typescript declaration instead of using jsdoc
README file update

q: what will cookies do in this project

### CODE REVIEW

File: src/LogglyApi.ts [hunk 1]
```diff
@@ -1,64 +1,34 @@
...
-
-    return promise;
-  }
-
-  _buildUrl() {
-    const endpoint = '//logs-01.loggly.com/inputs/';
-    return endpoint + this.token + '/tag/' + this.tags + '/';
+      .then(function (jsonData) {
+        return jsonData
+      })
```
Comment (rt-cff) [inline src/LogglyApi.ts]: Is this still necessary?

**Comment by glennfriend** [comment]
r+

File: src/LogglyLoggerFormatter.ts [hunk 2]
```diff
@@ -32,21 +21,21 @@ export class LogglyLoggerFormatter {
...
         host: window.location.hostname,
         path: window.location.pathname,
         query: window.location.search,
         url: window.location.href,
-      };
+      }
       if (document.referrer) {
-        json.url_info.referrer = document.referrer;
+        json.url_info.referrer = document.referrer
       }
```
Comment (rt-cff) [inline src/LogglyLoggerFormatter.ts:35]: I believe document is not defined in SSR too?

**Review by rt-cff — COMMENTED** [review]
r+ with comments

File: src/LogglyLoggerFormatter.ts (hunk 2 above)
Comment (TianyiLi) [reply src/LogglyLoggerFormatter.ts:35]: this block is under `isWeb` check

**Pull request merged**